src/uu/hashsum/src/hashsum.rs

```diff
@@ -370,6 +373,11 @@ pub fn uu_app_common<'a>() -> Command<'a> {
                 .long("tag")
                 .help("create a BSD-style checksum"),
         )
+        .arg(
+            Arg::new("no-names")
+                .long("no-names")
+                .help("Omits filenames in the output"),
```

Comment:
Maybe add something like
```suggestion
                .help("Omits filenames in the output (option not present in GNU/Coreutils)"),
```